Add debug logging instructions for certificate issues (#603)
NabuCasa/support: src/cloud/troubleshooting-remote-ui/remote-access-is-being-prepared.md

5. Once Home Assistant restarts, go back to the **Home Assistant Cloud** page and sign in with your Nabu Casa credentials.
6. Wait a few minutes while Home Assistant attempts to request and download a new SSL certificate for your remote URL.

If resetting cloud data **did not resolve the issue**, **do not reset it again**. Instead, enable debug logging to help diagnose the problem:

> 1. Add the following logger configuration to your `configuration.yaml` file:

Reviewer: ```suggestion
1. Add the following logger configuration to your [`configuration.yaml`](https://www.home-assistant.io/docs/configuration/) file:
```

   ```yaml
   logger:
     default: info
     logs:
       homeassistant.components.cloud: debug
       hass_nabucasa: debug
       acme: debug
       snitun: debug
   ```

2. Restart Home Assistant and allow it to run for approximately 15 minutes.

> 3. Download the Home Assistant core logs from [**Settings** > **System** > **Logs**](https://my.home-assistant.io/redirect/logs/).

Reviewer 2: ```suggestion
3. Download the Home Assistant Core logs from [**Settings** > **System** > **Logs**](https://my.home-assistant.io/redirect/logs/).
```

> 4. Use the 'Submit a ticket' button on this page and include the downloaded logs with your request. The debug information will help identify the root cause of your issue.

Reviewer: ```suggestion
4. Use the 'Submit a ticket' button on this page and include the downloaded logs with your request. This log will help identify the root cause of your issue.
```

> 4. Contact Nabu Casa support and include the downloaded logs with your support request. The debug information will help identify the root cause of the certificate generation issue.

Author: Agreed
